mlflow/tracking/fluent.py

```diff
@@ -613,6 +613,15 @@ def last_active_run() -> Optional[Run]:
     return get_run(last_active_run_id)
 
 
+def _get_latest_active_run():
```

Comment:
Can you update the name to indicate this is global scope, and add docstring that describes the difference from `last_active_run()`?

Reply:
Can we also update docstring of `last_active_run` to state it is thread-local, because it's user-facing API.

Reply:
I don't think `_get_latest_active_run` indicates global vs thread-local. What about `_get_global_active_run`?